src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -145,7 +145,7 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.AddTransient<VirtualFileSystem>();
             services.AddTransient<VirtualFileSystemMiddleware>();
 
-            if (SystemEnvironment.Instance.IsFlexConsumptionSku())
+            if (SystemEnvironment.Instance.IsLinuxConsumptionOnLegion())
```

Comment:
I might be missing something, but shouldn't this be 

`if(SystemEnvironment.Instance.IsFlexConsumptionSku() || SystemEnvironment.Instance.IsLinuxConsumptionOnLegion())`

?

Reply:
Oh yes fixed